tests/unit/test_data_classes.py

```diff
@@ -259,6 +259,25 @@ def test_base_proxy_event_get_query_string_value():
     assert value is None
 
 
+def test_base_proxy_event_get_multi_value_query_string_values():
+    default_values = ["default_1", "default_2"]
+    set_values = ["value_1", "value_2"]
+
+    event = BaseProxyEvent({})
+    values = event.get_multi_value_query_string_values("test", default_values)
+    assert values == default_values
+
+    event._data["multiValueQueryStringParameters"] = {"test": set_values}
+    values = event.get_multi_value_query_string_values("test", default_values)
+    assert values == set_values
+
+    values = event.get_multi_value_query_string_values("unknown", default_values)
+    assert values == default_values
+
+    values = event.get_multi_value_query_string_values("unknown")
+    assert values is None
```

Comment:
```suggestion
    assert values == []
```

changing to a default list to reduce boilerplate in customers code